lib/utils/assign.js

```diff
@@ -0,0 +1,3 @@
+// Aliasing Object.assign allows the minifier to shorten `Object.assign` to a single character variable.
+// Shortens 13 bytes to 1 at the cost of another allocation.
+export default Object.assign
```

Comment:
Iwonder if we stuck the polyfill code in here, and if Object.assign doesn't exist, we return our own object assign function?

The polyfill code is relatively small.:
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Polyfill

Similar to what you did with `Array.includes` (although slightly more code).